runtime/nls/j9gc/j9modron.nls

```diff
@@ -989,3 +989,22 @@ J9NLS_GC_OPTIONS_HIERARCHICAL_SCAN_ORDERING_NOT_SUPPORTED_WARN.system_action=The
 J9NLS_GC_OPTIONS_HIERARCHICAL_SCAN_ORDERING_NOT_SUPPORTED_WARN.user_response=Refer to the OpenJ9 documentation for -Xgc:hierarchicalScanOrdering.
 
 # END NON-TRANSLATABLE
+
+J9NLS_CHECKPOINTGCTHREAD_VALUE_USER_MUST_BE_AT_MOST_GCTHREAD_WARN=-XX:CheckpointGCThreads will be ignored. The specified value %zu must not be larger than user specified value of the total GC threads %zu using -Xgcthreads, -Xgcmaxthreads or -Xgc:threads
+# START NON-TRANSLATABLE
+J9NLS_CHECKPOINTGCTHREAD_VALUE_USER_MUST_BE_AT_MOST_GCTHREAD_WARN.explanation=-XX:CheckpointGCThreads must be smaller than or equal to the number of gc threads
+J9NLS_CHECKPOINTGCTHREAD_VALUE_USER_MUST_BE_AT_MOST_GCTHREAD_WARN.system_action=The JVM ignores the -XX:CheckpointGCThreads option.
+J9NLS_CHECKPOINTGCTHREAD_VALUE_USER_MUST_BE_AT_MOST_GCTHREAD_WARN.user_response=Refer to the OpenJ9 documentation and adjust -Xgcthreads/-XX:CheckpointGCThreads to make sure number of gc threads is larger than or equal to the number of checkpointgcthreads
+J9NLS_CHECKPOINTGCTHREAD_VALUE_USER_MUST_BE_AT_MOST_GCTHREAD_WARN.sample_input_1=4
+J9NLS_CHECKPOINTGCTHREAD_VALUE_USER_MUST_BE_AT_MOST_GCTHREAD_WARN.sample_input_2=4
+
+# END NON-TRANSLATABLE
+
+J9NLS_CHECKPOINTGCTHREAD_VALUE_HEURISTIC_MUST_BE_AT_MOST_GCTHREAD_WARN=-XX:CheckpointGCThreads will be ignored. The specified value %zu must not be larger than heuristic calculated value of the total GC threads %zu
```

Comment:
minor, but 'heuristic' word should be closer to 'gcthreads' and perhpas use 'calculated' instead, since it's an adjective

J9NLS_CHECKPOINTGCTHREAD_VALUE_MUST_BE_AT_MOST_CALCULATED_GCTHREAD_VALUE_WARN

Reply:
I think we don't need `heuristic calculated`, just `calculated`